CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ### Features
 
 - `[pretty-format]` Added support for serializing custom elements (web components) ([#10217](https://github.com/facebook/jest/pull/10237))
+- `[jest-each, jest-jasmine, jest-circus]` Add support for .concurrent.each ([#9326](https://github.com/facebook/jest/pull/9326))
```

Comment:
merge in master and make sure the entry is in the correct place 🙂

Reply:
Move it up